src/config/bookconfig.rs

```diff
@@ -5,7 +5,7 @@ use super::tomlconfig::TomlConfig;
 use super::jsonconfig::JsonConfig;
 
 /// Configuration struct containing all the configuration options available in mdBook.
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
`BookConfig` is never serialized or deserialized directly (only through filling the fields ``)`BookConfig` is never directly serialized nor deserialized. This is done by copying fields to/from `TomlConfig`. So I've decided to clean it up to avoid misconception that it actually is.

I should probably find other instances of such spurious derives.